src/Cli/UpdateTerminusDocsCommands.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+namespace UpdateTool\Cli;
+
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerAwareTrait;
+use Robo\Common\ConfigAwareTrait;
+use Robo\Contract\ConfigAwareInterface;
+use Hubph\HubphAPI;
+use Hubph\Git\WorkingCopy;
+use Hubph\Git\Remote;
+
+/**
+ * Commands used to interact with Terminus releases.
+ */
+class UpdateTerminusDocsCommands extends \Robo\Tasks implements ConfigAwareInterface, LoggerAwareInterface
+{
+    use ConfigAwareTrait;
+    use LoggerAwareTrait;
+
+    /**
+     * Update terminus commands and releases in documentation repo.
+     *
+     * @command terminus:update-docs
+     */
+    public function terminusUpdateDocs($options = [
+        'as' => 'default',
+        'update-commands' => true,
+        'update-releases' => true,
+        'terminus-release' => 'latest',
+        'github-repo' => 'pantheon-systems/documentation',
+        'terminus-repo' => 'pantheon-systems/terminus',
+        'base-branch' => 'main',
+        'branch-name-prefix' => 'docs-update-terminus-',
+        'commit-message' => 'Update terminus information.',
+        'pr-body' => '',
+        'pr-title' => '[UpdateTool - Terminus Information] Update commands and releases.',
+        'dry-run' => false,
+    ])
+    {
+        $api = $this->api($options['as']);
+        $updateCommands = $options['update-commands'];
+        $updateReleases = $options['update-releases'];
+        if (!$updateCommands && !$updateReleases) {
+            throw new \Exception("Either commands or releases must be updated.");
```

Comment:
`"Only commands or releases can be updated."`?

Reply:
Existing error message is clearer; this only happens if the user runs with both `--no-update-commands` and `--no-update-releases`.

Reply:
How about `"You must update at least one of commands or releases."`?

Reply:
Given that the user action is to exclude rather than include actions, I might say "Both --no-update-commands and --no-update-releases specified; nothing to do."

Reply:
Fixed wording as Greg suggested